src/ProductCard/index.tsx

```diff
@@ -25,6 +27,11 @@ interface Props {
   matchPercentage?: number
   /** discounted price or range of discounted prices if any */
   discountPrice?: number[]
+  storeLocation?: string
+  productPage?: string
+  deliveryDate?: string
+  financeLink?: string
+  storeLocationBtnOnClick?: Function
```

Comment:
What are the arguments of the Function? What is being returned?
Better to do this: 
```
() => void
```
You can put the arguments and their types as well as what the return type will be